packages/gcc7.rb

```diff
@@ -125,8 +130,29 @@ def self.install
       
       # Add a compatibility symlink to enable building programs with Link Time Optimization (LTO)
       system "ln -sfv #{CREW_DEST_PREFIX}/libexec/gcc/$(gcc -dumpmachine)/7.3.0/liblto_plugin.so #{CREW_DEST_LIB_PREFIX}/bfd-plugins/"
-      system "ln -sv gcc-7.3 #{CREW_DEST_PREFIX}/bin/gcc"
-      system "ln -sv gcc-7.3 #{CREW_DEST_PREFIX}/bin/cc"
+      
+      # Make symbolic links
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gcc-7.3 #{CREW_DEST_PREFIX}/bin/cc"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gcc-7.3 #{CREW_DEST_PREFIX}/bin/gcc"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/c++-7.3 #{CREW_DEST_PREFIX}/bin/c++"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/g++-7.3 #{CREW_DEST_PREFIX}/bin/g++"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/cpp-7.3 #{CREW_DEST_PREFIX}/bin/cpp"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gcc-ar-7.3 #{CREW_DEST_PREFIX}/bin/gcc-ar"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gcc-nm-7.3 #{CREW_DEST_PREFIX}/bin/gcc-nm"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gcc-ranlib-7.3 #{CREW_DEST_PREFIX}/bin/gcc-ranlib"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gcov-7.3 #{CREW_DEST_PREFIX}/bin/gcov"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gcov-dump-7.3 #{CREW_DEST_PREFIX}/bin/gcov-dump"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gcov-tool-7.3 #{CREW_DEST_PREFIX}/bin/gcov-tool"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gfortran-7.3 #{CREW_DEST_PREFIX}/bin/gfortran"
+      
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-c++-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-c++"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-g++-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-g++"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ar-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ar"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-nm-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-nm"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ranlib-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ranlib"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gfortran-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gfortran"
```

Comment:
The source of all the symlinks should be `#{CREW_PREFIX}` instead of `#{CREW_DEST_PREFIX}`.  Target is fine.

Reply:
Change all #{CREW_DEST_PREFIX} to #{CREW_PREFIX} ?

Reply:
Only the source.  Target is fine.  Example:
```
system "ln -sv #{CREW_PREFIX}/bin/$(gcc -dumpmachine)-gfortran-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gfortran"
```

Reply:
@uberhacker fixed